src/main/java/lbot/Launcher.java

```diff
@@ -0,0 +1,18 @@
+package lbot;
+
+import javafx.application.Application;
+import lbot.gui.Main;
+
+/**
+ * LStart point of LBot.
```

Comment:
Super minor nit here, but might this be a typo in your header comment or a style choice in saying `LStart point`, as in the same branding as `LBot` perhaps?

Reply:
hey thanks! didn't catch that mistake, will fix it soon :)